java/client/src/test/java/glide/api/GlideClientTest.java

```diff
@@ -1581,6 +1582,30 @@ public void invokeScript_with_ScriptOptionsGlideString_returns_success() {
         assertEquals(payload, response.get());
     }
 
+    @SneakyThrows
+    @Test
+    public void scriptShow_returns_script_source_glidestring() {
+        // setup
+        GlideString scriptSource = gs("return { KEYS[1], ARGV[1] }");
+        Script script = mock(Script.class);
```

Comment:
Remove
Your unit tests are probably failing because of this mock.  And script is not needed anywhere.